state_of_css.yml

```
@@ -97,7 +122,21 @@ translations:
     - key: sections.environments.title
       t: Prostředí
     - key: sections.environments.description
-      t: Pro jaká prostředí píšete CSS?
+      t: Jaká prostředí a problémy s přístupností zvažujete při psaní CSS?
+
+    - key: sections.tools_others.title
+      t: Ostatní nástroje
+    - key: sections.tools_others.description
+      t: Ostatní CSS nástroje.
+
+    - key: sections.usage_css.title
+      t: Používání CSS
+    - key: sections.usage_css.description
+      t: Jak používáte CSS.
+    - key: sections.resources_css.title
+      t: Zdroje
+    - key: sections.resources_css.description
+      t: Jaké zdroje CSS konzultujete?
```

I'd prefer something like:

```suggestion
      t: V jakých zdrojích o CSS vyhledáváte?
```

Sounds more czech than _to consult sth._